SparseGrids/TasmanianSparseGridWrapC.cpp

```diff
@@ -108,6 +108,16 @@ void tsgMakeFourierGrid(void *grid, int dimensions, int outputs, int depth, cons
     if (depth_type == type_none){ depth_type = type_level; }
     ((TasmanianSparseGrid*) grid)->makeFourierGrid(dimensions, outputs, depth, depth_type, anisotropic_weights, limit_levels);
 }
+void tsgMakeGridFromCustomTabulated(void *grid, int dimension, int outputs, int depth, const char *sType, void *custom_tabulated,
+                                    const int *anisotropic_weights, const int *limit_levels) {
+    TypeDepth depth_type = IO::getDepthTypeString(sType);
+    #ifndef NDEBUG
+    if (depth_type == type_none){ cerr << "WARNING: incorrect depth type: " << sType << ", defaulting to type_iptotal." << endl; }
+    #endif // NDEBUG
+    ((TasmanianSparseGrid*) grid)->makeGlobalGrid(dimension, outputs, depth, depth_type,
+                                                  std::move(*reinterpret_cast<TasGrid::CustomTabulated*>(custom_tabulated)),
```

Comment:
If you move here, that would leave `custom_tabulated` in an undefined state. While C++ knows how to handle the problem, Python will try to issue a delete command on the pointer (when calling the destructor of CustomTabulated and then the C/C++ method).

The safest option is to use a copy.

Reply:
Done.